x-pack/plugins/apm/server/routes/services.ts

```diff
@@ -46,6 +48,36 @@ export const servicesRoute = createRoute({
   },
 });
 
+export const serviceDetailsRoute = createRoute({
+  endpoint: 'GET /api/apm/services/{serviceName}',
```

Comment:
I'm not sure this is the right route. I think both the icon info (before clicking popover) and the details (after clicking popover) should be grouped under the same route prefix. 

Something like:
'GET /api/apm/services/{serviceName}/metadata/icons'

and

'GET /api/apm/services/{serviceName}/metadata/details'

Reply:
thanks for the suggestion, I was also not comfortable with the names I chose!!